components/base/sensorcontrolled/velocities.go

```diff
@@ -43,7 +52,7 @@ func (sb *sensorBase) SetVelocity(
 // startControlLoop uses the control config to initialize a control loop and store it on the sensor controlled base struct.
 // The sensor base is the controllable interface that implements State and GetState called from the endpoint block of the control loop.
 func (sb *sensorBase) startControlLoop() error {
-	loop, err := control.NewLoop(sb.logger, sb.controlLoopConfig, sb)
+	loop, err := control.NewLoop(sb.logger, *sb.controlLoopConfig, sb)
```

Comment:
Again, we should keep an eye on our pointers for controls-related changes. I suppose we'll tackle this in John's reactors. https://medium.com/beyn-technology/should-we-use-pointers-in-go-truth-is-complicated-75fccae92f27

Nick S also has an excellent engineering talk about this topic.

Reply:
I don't remember talking about pointers in controls, I'll read this article and look at the engineering talk and then we can discuss if there's better options during a refactor.